App.html

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="UTF-8">
+        <title>Innocent's Portfolio</title>
+        <link rel="stylesheet" href="App.css">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+        <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@700&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link href="https://fonts.googleapis.com/css2?family=Crete+Round&display=swap" rel="stylesheet">
+        <link rel="preconnect" href="https://fonts.googleapis.com">
+        <link href="https://fonts.googleapis.com/css2?family=Poppins&display=swap" rel="stylesheet">
+    </head>
+    <body class="container">
+        <header class="header">
+            <nav class="nav">
+                <h1 class="nav-title">Innocent N.</h1>
+                <a href="#">
+                    <img src="./images/menu.png" alt="menu-icon" class ="nav-menu"/>
+                </a>      
+            </nav>
+        </header>
+        <section class="hero">
+            <div class="main">
+                <h1 class="main-header">Hey There.<br>I'm Innocent</h1>
+                <h2 class="sub-header">I'm a Software Developer</h2>
+                <p class="main-paragraph">I can help you build a product,
+                    feature or website Look through some of my work and experience!
+                    If you like what you see and have a project you need coded, 
+                    don't hestitate to contact me.</p>
+                <ul class="social-media">
```

Comment:
- [ ] Please make sure all your social icons are the same size, and should like this given design:
![image](https://user-images.githubusercontent.com/92215878/219729115-bca3efe2-8345-49cc-ac1c-d41e1f83a967.png)

_Consider exporting them again from Figma_